app/controllers/api/tasks_controller.rb

```diff
@@ -16,9 +16,14 @@ def update
     if @task.users.first.id != params[:user_id]
       self.unassign(@task.users.first.id, @task.id)
       self.assign(params[:user_id], @task.id)
+    end
+    user = User.find(params[:current_user])
+    if @task.completed_status == 0
+      tasks = user.tasks.where(:completed_status => 0).order(:due_date)
     else
-      render(:json => {:message => 'Task successfully updated!'}.to_json)
+      tasks = user.tasks.where(:completed_status => 1).order(:updated_at).reverse_order
```

Comment:
you should be able to do `.order(desc: :updated_at)` instead of using two queries

https://apidock.com/rails/ActiveRecord/QueryMethods/order